src/cobald/daemon/runners/asyncio_watcher.py

```diff
@@ -20,13 +20,17 @@ def asyncio_main_run(root_runner: BaseRunner):
     """
     Create an ``asyncio`` event loop running in the main thread and watching runners
 
-    Using ``asyncio`` to handle suprocesses requires a specific loop type to run in the main thread.
+    Using ``asyncio`` to handle suprocesses requires a specific loop type
```

Comment:
I think you mean subprocesses?

```suggestion
    Using ``asyncio`` to handle subprocesses requires a specific loop type
```